packages/react-charts/src/components/ChartAxis/ChartAxis.tsx

```diff
@@ -12,7 +12,7 @@ import {
   RangePropType,
   ScalePropType,
   StringOrNumberOrList,
-  TickLabelProps
+  LabelProps
```

Comment:
https://github.com/FormidableLabs/victory/commit/439e0ad8de23d1d6ed5580a90f60feaca3f09c4e#diff-1d0ded8dfdfbe48d50e6a99272c7548f81143f224f88b6528023cc87a32e9a40R129-R132.
Here they uses `LabelProps` to replace the `TickLabelProps` in Victory.